### PR TITLE
intro edits, chevrette 1

### PR DESCRIPTION
**Pull request by chevrm**
**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [x] Yes, I added myself or am already a contributor

**Any more details?**

### CODE REVIEW

File: content/02.intro.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ## Introduction
 
-Deep learning (DL), a subfield of machine learning (ML) that focuses on deep artificial neural networks, is exploding in popularity and is increasingly being used for biological data analysis [@doi:10.1098/rsif.2017.0387].
-However, DL itself remains an active area of research, and the complexity of this field poses a large barrier of entry to those who would like to correctly utilize state-of-the-art DL technology in biological research applications.
-While much has been written about the impressive results of DL in biology, there is a comparative dearth of literature articulating best practices for its use, with most instructional literature focusing on ML, rather than DL [@doi:10.1186/s13040-017-0155-3].
+Deep learning (DL), a subfield of machine learning (ML) implementing hierarchical artificial neural networks, is increasingly used for the characterization of genomic and biological data [@doi:10.1098/rsif.2017.0387].
```
[the remark this answers is not in the document]
Comment (AlexanderTitus) [reply content/02.intro.md]: "Characterization" is often used in biology literature, but it may be better to keep it at the more common vernacular since these are tips that hopefully cross fields. 

I would just say biological data. This encompasses genomic data but does not limit the applicability of these tips to just genomic data.

[the remark this answers is not in the document]
Comment (rasbt) [reply content/02.intro.md]: I would also favor "analysis" because characterization may be more like "describing" than "predicting" (usually ML/DL methods are used for various prediction tasks in bio as far as I know). E.g., in a drug discovery context, I am thinking of DL for predicting active molecules as drug candidates, and molecular dynamics simulations and wet-lab experiments would then be used to characterize the binding mechanism or sth like that. Not sure about genomics-specific work though

File: content/02.intro.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+Despite its growing popularity, DL itself remains an active area of research, and its everchaning complexity and lack of current beginner resources pose large barriers of entry to newcomers who wish to utilize state-of-the-art DL in their research.
```
[the remark this answers is not in the document]
Comment (rasbt) [reply content/02.intro.md]: Afaik there's tons and tons of beginner resources for DL (more than for most other field/methods). Maybe we should rephrase this a bit to specifically mean DL applications to biological problems, because there, beginner resources are indeed a bit scarce and practically non-existent aside from short method sections in research papers

**Comment by chevrm** [comment]
Adjustments made to address comment. Should be good to go.

**Comment by Benjamin-Lee** [comment]
Can someone approve/request changes to this?

File: content/02.intro.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 ## Introduction
 
-Deep learning (DL), a subfield of machine learning (ML) that focuses on deep artificial neural networks, is exploding in popularity and is increasingly being used for biological data analysis [@doi:10.1098/rsif.2017.0387].
-However, DL itself remains an active area of research, and the complexity of this field poses a large barrier of entry to those who would like to correctly utilize state-of-the-art DL technology in biological research applications.
-While much has been written about the impressive results of DL in biology, there is a comparative dearth of literature articulating best practices for its use, with most instructional literature focusing on ML, rather than DL [@doi:10.1186/s13040-017-0155-3].
+Deep learning (DL), a subfield of machine learning (ML) implementing hierarchical artificial neural networks, is increasingly used for the analysis of biological data [@doi:10.1098/rsif.2017.0387].
```
Comment (rasbt) [inline content/02.intro.md]: >implementing hierarchical artificial neural networks,

I am confused by the term "hierarchical artificial neural networks." I think this is way to specific. I think we should use sth like "artificial neural networks with many layers" or "deep artificial neural networks" is better. (hierarchical artificial neural networks refers to a special type of architecture, like these tree-structured neural nets that deep mind used for AlphaGo if I remember correctly)

File: content/02.intro.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
+Biological insight garnered from DL has been well-documented in the scientific literature, with applications ranging from predicting protein-drug binding kinetics [@doi:10.1038/s41467-017-02388-1] to identifying the lab-of-origin of synthetic DNA [@doi:10.1038/s41467-018-05378-z]. However, few resources articulate DL best practices to the scientific community. Most instructional literature focuses on ML broadly, rather than DL specifically, further limiting accessibility and reproducibility [@doi:10.1186/s13040-017-0155-3].
 
-To fix this problem, we solicited input from a diverse community of researchers, who wrote this manuscript collaboratively using the GitHub version control platform [@url:https://github.com/Benjamin-Lee/deep-rules] and Manubot [@url:https://greenelab.github.io/meta-review/].
+To address this issue, we solicited input from a diverse community of researchers, who wrote this manuscript collaboratively using the GitHub version control platform [@url:https://github.com/Benjamin-Lee/deep-review] and Manubot [@url:https://greenelab.github.io/meta-review/].
```
Comment (Benjamin-Lee) [inline content/02.intro.md]: Why the change from `deep-rules` to `deep-review`?

Comment (rasbt) [inline content/02.intro.md]: We shouldn't start a new paragraph with "this issue" without defining "this." On the flipside, we don't want to sound too repetitive. To resolve this, I think this sentence can just be added at the end of the previous paragraph -- would fit nicely there imho

Comment (Benjamin-Lee) [reply content/02.intro.md]: Done

**Comment by rasbt** [comment]
LGTM now!
